File: Database/Assembly_SRPG/TipsParam.cs
namespace SRPG
{
    using System;

    public class TipsParam
    {
        public string iname;
        public ETipsType type;
        public int order;
        public string title;
        public string text;
        public string[] images;
        public bool hide;
        public string cond_text;

        public TipsParam()
        {
            base..ctor();
            return;
        }

        public void Deserialize(JSON_TipsParam json)
        {
            this.iname = json.iname;
            this.type = (int) Enum.ToObject(typeof(ETipsType), json.type);
            this.order = json.order;
            this.title = json.title;
            this.text = json.text;
            this.images = json.images;
            this.hide = (json.hide == 0) == 0;
            this.cond_text = json.cond_text;
            return;
        }
    }
}

File: Database_V2/ParamFunctions/TipsParam.py
from ParamFunctions._variables import ENUM
def TipsParam(json):
    this={}#TipsParamjson)
    if 'iname' in json:
        this['iname'] = json['iname']
    if 'type' in json:
        this['type'] = ENUM['ETipsType'][json['type']]
    if 'order' in json:
        this['order'] = json['order']
    if 'title' in json:
        this['title'] = json['title']
    if 'text' in json:
        this['text'] = json['text']
    if 'images' in json:
        this['images'] = json['images']
    if 'hide' in json:
        this['hide'] = json['hide']!=0
    if 'cond_text' in json:
        this['cond_text'] = json['cond_text']
    return this
